R/mboostLSS.R

```diff
@@ -159,16 +198,27 @@ mboostLSS_fit <- function(formula, data = list(), families = GaussianLSS(),
         ## update value of nuisance parameters in families
         for (k in mods[-j]){
             if (!is.null(fit[[k]]))
-                assign(names(fit)[k], fitted(fit[[k]], type = "response"),
+                assign(names(fit)[k], families[[k]]@response(mboost:::fitted.mboost(fit[[k]])),
```

Comment:
Is it possible to use `assign(names(fit)[k], fitted(fit[[k]], type = "response"),`?